target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -246,6 +246,11 @@ mtc,wr1201)
 mzk-ex750np)
 	set_wifi_led "$boardname:red:wifi"
 	;;
+netgear,r6020)
+	ucidef_set_led_switch "lan" "lan" "$boardname:green:lan" "switch0" "0x0f"
+	ucidef_set_led_wlan "wlan2g" "WiFi 2.4GHz" "$boardname:green:wlan2g" "phy0tpt"
+	ucidef_set_led_wlan "wlan5g" "WiFi 5GHz" "$boardname:green:wlan5g" "phy1tpt"
```

Comment:
@crdotson Drop the wlan2g/wlan5g lines and use `linux,default-trigger = "phy0tpt";` and `linux,default-trigger = "phy1tpt";` in your LED nodes instead. Configuration of LEDs in DTS is preferred. Example:

https://github.com/openwrt/openwrt/blob/e65f935de9e67ea07a251f114455f12e9555c5bf/target/linux/ath79/dts/ar9344_comfast_cf-e120a-v3.dts#L36-L40